### ISSUE TITLE
Update first email copy

### ISSUE THREAD

**Issue by jasonlally** [sfbrigade/adopt-a-drain#90]
Jean's updates here: https://docs.google.com/document/d/1ogprm41vY17Wj-rSjOds7UOJDQGtv6t4oiswYEDTtAg/edit

**Comment by jszwedko** [comment]
Closed by #95

**Comment by jeanwalshie** [comment]
1) the hyperlink to Twitter at the bottom of the email is not working (http://twitter.com/sfwater) 
2) there were spacing issues in the copy and other minor issues that i fixed in the Google doc.
I recommend that we replace the whole email with the current copy.

**Comment by jszwedko** [comment]
@jeanwalshie thanks for the updates! I replaced the text with what is currently in the Google doc.

Note: we are currently sending plaintext e-mails (as opposed to HTML), so I had to modify the copy slightly to change:

> If not, report it to 311.

where "report it to 311" was a hyperlink, to

> If not, report it to 311 at http://www.sf311.org/index.aspx.

I'd like to avoid sending HTML e-mails as getting them "right" so that they render properly in all clients can sometimes be a bit of a pain and the only "feature" we'd currently be using is to create hyperlinks out of URLs which most email clients already do for the user anyway. If it is a hard requirement, we can revisit.
